docs/networking/cilium.md

```diff
@@ -85,7 +85,7 @@ Then enable etcd as kvstore:
 As of kOps 1.19, BPF NodePort is enabled by default for new clusters if the kubernetes version is 1.12 or newer. It can be safely enabled as of kOps 1.18.
 
 In this mode, the cluster is fully functional without kube-proxy, with Cilium replacing kube-proxy's NodePort implementation using BPF.
-Read more about this in the [Cilium docs - kubeproxy free](https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/) and [Cilium docs - NodePort](https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/#nodeport-devices)
+Read more about this in the [Cilium docs - kubeproxy free](https://docs.cilium.io/en/v1.12/gettingstarted/kubeproxy-free/) and [Cilium docs - NodePort](https://docs.cilium.io/en/v1.12/gettingstarted/kubeproxy-free/#nodeport-devices)
```

Comment:
```suggestion
Read more about this in the [Cilium docs - kubeproxy free](https://docs.cilium.io/en/stable/network/kubernetes/kubeproxy-free/) and [Cilium docs - NodePort](https://docs.cilium.io/en/stable/network/kubernetes/kubeproxy-free/#nodeport-devices)
```

Reply:
Could do these as `1.13` links instead to get latest permalinks

Reply:
@hakman could you re-review this?

Per my opening comment, this PR:
> follows the convention in the rest of this doc to use versions

Per my previous comments above, `stable` is not a permalink, so I do not think that is ideal and does not follow the existing convention in these docs

Reply:
OK, but then at least let's be consistent and use v1.12 everywhere.

Reply:
@hakman found the v1.13 links (which is latest right now) for all of these and updated them all to v1.13 👍